pwiz_tools/BiblioSpec/src/DiaNNSpecLibReader.cpp

```diff
@@ -476,9 +476,15 @@ class Library {
         in.read((char*)&iRT_min, sizeof(double));
         in.read((char*)&iRT_max, sizeof(double));
         read_array(in, entries, version);
+        int i = 0;
```

Comment:
This is a good idea but I'll refactor it to use iterators (with a pre-check that the vectors are the same length).